### DISCUSSION TITLE
[Features]: Post BeReals, RealMojis, comments directly from BeUnblurred and some other feature request.

### DISCUSSION THREAD

**Discussion by stepersy** [macedonga/beunblurred#26] [Ideas]
Hi, first of all, thank you for your great work. I have used several "BeReal Clients" but none as good as yours. I love the design and ease of use and I love the friends of friends feature!

I would like to suggest a few features, just to make this client THE CLIENT. Some are already available in other "BeReal" clients like BeFake or TooFake, some are not. These features are:

1. The ability to post a BeReal directly from the website, using your own photos from the gallery, or even GIFs, with custom location support and 0 retakes (or even the ability to post it on time even if I'm late). Most of these features are available in BeFake by rvaidun or TooFake.
2. The ability to add comments and RealMojis directly from the site (BeFake)
3. An APK or a link to the Play Store. The website says that there's a version of the website as an app in the Play Store, but the Play Store gives an error when trying to reach the link. Probably better to keep it on GitHub or even F-Droid.

These are some of my suggestions. I know they are a lot, but if you can add just some of them it would be amazing and it would become the perfect, nicest BeFake client.

Thanks in advance for your patience and work. Keep it up!

E Forza Italia! 🇮🇹

**Comment by macedonga** [comment]
Hey, thanks for the kind words!
I decided not to let users post fake BeReals from the website because I didn't want to go *too much* against BeReal's ToS. Reading the terms of service now, it looks like it's against the rules to post custom BeReals, but they don't mention anything about accessing the social through custom clients. I guess letting users comment and react to BeReals wouldn't be against their ToS though, so I might implement that ;-)
Unfortunately BeReal's lawyer sent a Trademark Complaint to Google, so I had to take the app down. That complaint is 100% not legally applicable, but since they know I am not able to legally fight back their claim, they sent it anyways... So yeah, I might just change the link to an APK file, or upload it on F-Droid, even if I think it'll get taken down again. If you want though, you can install the application as a PWA anyways, if you're using BeUnblurred on Android.
